core/src/test/scala/org/apache/spark/serializer/SerializationDebuggerSuite.scala

```diff
@@ -205,6 +214,12 @@ class SerializableClass1 extends Serializable
 
 class SerializableClass2(val objectField: Object) extends Serializable
 
+class SerializableClassWithStringException(val objectField: Object) extends Serializable {
+  override def toString: String = {
```

Comment:
This simulate the behavior of `toString` implementation of `TreeNode`, since `SQLConf.get` may throw exception

Reply:
Yes, it only throw exception in during test